package.json

```diff
@@ -118,6 +119,8 @@
     "@types/node": "^17.0.30",
     "@types/nodemailer": "^6.4.4",
     "@types/react": "^17.0.44",
+    "@types/react-beautiful-dnd": "^13.1.3",
+    "@types/selenium-webdriver": "^4.0.16",
```

Comment:
We don't use the `selenium-webdriver` package anymore so don't think this is needed